core/src/main/java/org/elasticsearch/action/ingest/IngestDisabledActionFilter.java

```diff
@@ -52,7 +52,7 @@ public int order() {
     }
 
     private static void failRequest(String pipelineId) {
-        throw new IllegalArgumentException("ingest plugin is disabled, cannot execute pipeline with id [" + pipelineId + "]");
```

Comment:
I know this behavior was already existing, but why don't we forward the request like we do eg master requests on non master nodes.